stable/scaler/values.yaml

```diff
@@ -2,17 +2,43 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# kubeconfig: /home/iguazio/.kube/config
+
 scaler:
   image:
-    repository: iguaziodocker/scaler
-    tag: 1.0.0
+    repository: iguaziodocker/autoscaler
+    tag: 0.1.0
     pullPolicy: IfNotPresent
+  
+  config:
+    command: ./autoscaler
```

Comment:
Place the `command` in the `image` section.
Can the command be without the `./` prefix? Maybe you should place it in the container `PATH`